### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Pratishah**
added profile icon and login/signup button , added test cases 

Ticket Link: https://abhirams.atlassian.net/browse/P3ECWFC-12

Screenshot (25)
Screenshot (27)

### CODE REVIEW

File: src/components/Navbar/TopNavbar.tsx [hunk 1]
```diff
@@ -1,39 +1,119 @@
...
+      localStorage.setItem("isLoggedin", "false");
+    } else {
+      localStorage.setItem("isLoggedin", "true");
+      setisLoggedin(true);
+    }
+  };
+  const isloggedin: boolean = isLoggedin;
+  isloggedin === true
+    ? (profilepicture = loggedProfile)
+    : (profilepicture = UnknownProfile);
```
Comment (pashedin) [inline src/components/Navbar/TopNavbar.tsx]: Non actionable: this logic could have been written in the src of the profile image itself, but its okay here for now, no need to change.

File: src/components/Navbar/__test__/TopNavbar.test.tsx [hunk 2]
```diff
@@ -49,9 +61,20 @@ describe("Navbar", () => {
...
+    if (isloggedin === "true") {
+      // eslint-disable-next-line jest/no-conditional-expect
+      expect(ProfileElement).toBeInTheDocument();
+      // eslint-disable-next-line jest/no-conditional-expect
+      expect(ProfileElement).toHaveAttribute("src", "man.png");
+    } else {
+      // eslint-disable-next-line jest/no-conditional-expect
+      expect(ProfileElement).toHaveAttribute("src", "profile.svg");
+    }
+
```
Comment (pashedin) [inline src/components/Navbar/__test__/TopNavbar.test.tsx]: we also need a counter test case which has isLoggedIn false

**Review by pashedin — CHANGES_REQUESTED** [review]
need one more tes case thi is done post that.

File: src/components/Navbar/__test__/TopNavbar.test.tsx (hunk 2 above)
Comment (Pratishah) [reply src/components/Navbar/__test__/TopNavbar.test.tsx]: added test for not loggedin case

**Review by pant-manas — COMMENTED** [review]
Please resolve the conflicts

File: src/components/Navbar/TopNavbar.tsx [hunk 3]
```diff
@@ -1,29 +1,38 @@
...
+  //   return false;
+  // });
+  const dispatch = useDispatch();
   let profilepicture;
 
   const isloginHandler = () => {
-    if (isloggedin === true) {
-      setisLoggedin(false);
-      localStorage.setItem("isLoggedin", "false");
+    console.warn("clicked", isLoggedin);
```
Comment (kboodi) [inline src/components/Navbar/TopNavbar.tsx]: remove console log

File: src/components/Navbar/TopNavbar.tsx [hunk 4]
```diff
@@ -1,29 +1,38 @@
...
+    if (isLoggedin === LoginStatus.LoginSuccess) {
+      dispatch(logout());
+      console.warn("after dispatch", isLoggedin);
```
Comment (kboodi) [inline src/components/Navbar/TopNavbar.tsx]: remove console

**Review by kboodi — APPROVED** [review]

**Review by pashedin — APPROVED** [review]

**Pull request merged**